apps/ledger-live-desktop/src/newArch/AnalyticsOptInPrompt/hooks/useCommonLogic.tsx

```diff
@@ -31,18 +42,27 @@ export const useAnalyticsOptInPrompt = ({ entryPoint }: Props) => {
   );
 
   useEffect(() => {
-    const isFlagEnabled = lldAnalyticsOptInPromptFlag?.enabled && !hasSeenAnalyticsOptInPrompt;
+    const isFlagEnabled =
+      lldAnalyticsOptInPromptFlag?.enabled &&
+      (!hasSeenAnalyticsOptInPrompt || entryPoint === EntryPoint.onboarding);
```

Comment:
You can transform that with useMemo instead of using useEffect + useState , like :

```suggestion
 const isFlagEnabled = useMemo(()=>
      lldAnalyticsOptInPromptFlag?.enabled &&
      (!hasSeenAnalyticsOptInPrompt || entryPoint === EntryPoint.onboarding),[lldAnalyticsOptInPromptFlag,hasSeenAnalyticsOptInPrompt,entryPoint])
```